src/components/atoms/ChatMessage/ChatMessage.scss

```diff
@@ -96,7 +96,9 @@ $reply-icon-radius: 20px;
   }
 
   &__reply-icon {
-    display: none;
+    color: white;
+    display: flex;
+    opacity: 0;
```

Comment:
Why changed from `display: none`? Why using `opacity` here?

Reply:
But surely we have it set to display: none in that instance because it explicitly shouldn’t be focusable, no?

Reply:
@0xdevalias I guess it's correct. People with accessibility issues will not know of the existence of this button, if it's display: none

Reply:
Isn’t that the point though? And then presumably it’s set to a visible state when it is meant to be known about, to both sighted and screen reader based users?